README.md

```diff
@@ -142,6 +165,10 @@ One of the following is required.
   `metadata` which should contain the name of your new lock and the contents you
   would like in the lock, respectively.
 
+> Note: all `out` operations described above will emit a git commit.
```

Comment:
`s/emit/generate/`. Rationale: "emit" means something active like a signal, while here there is nothing active.

Reply:
[github-fu] You can attach a patch for proposing changes in a PR. Makes the fixups processing easier for the branch owner (See David's suggestion from this morning, for example).